lib/chef/azure/commands/enable.rb

```diff
@@ -29,21 +29,19 @@ def initialize(extension_root, *enable_args)
   def run
     load_env
 
-    report_heart_beat_to_azure(AzureHeartBeat::NOTREADY, 0, "Enabling chef-service...")
-
     enable_chef
 
     if @exit_code == 0

```

Comment:
Is this the Scheduled Task exit code or the Chef Client exit code?  If the latter do we need to take account of the possible exit codes in https://github.com/chef/chef-rfc/blob/master/rfc062-exit-status.md#exit-codes-in-use

Reply:
This is exit code for chef service or chef task failure because this code gets executed after call to this method https://github.com/chef-partners/azure-chef-extension/blob/master/lib/chef/azure/commands/enable.rb#L62-L90 in which if there is any failure in enabling chef service or chef task we are setting exit code to 1 else its 0